docker-compose.yml

```diff
@@ -81,6 +81,7 @@ services:
     volumes:
       - ./etc/delfin:/etc/delfin
       - db_data:/var/lib/delfin
+      - metrics_dir:${DELFIN_METRICS_DIR}
```

Comment:
do we need to add this a separate volume . when you copy /etc/delfin to ./etc/delfin this config is already copied.. and exporter has the code to create directory as in conf..